lib/originals-helper.js

```diff
@@ -29,7 +29,7 @@ function nodeInIDLSpecs(node) {
  * @return {boolean} whether the namespace is global
  */
 function isGlobal(node) {
-  return nodeInTSDefs(node) || nodeInIDLSpecs(node);
+  return nodeInIDLSpecs(node) || nodeInTSDefs(node);
```

Comment:
Why change this?

Reply:
I wanted to guard against the possibility that a Web type exists in the IDL but the nodes for the TypeScript check don't exist, which would throw an error. I could try / catch it also, but this seemed easier

Reply:
The function shouldn't throw an error if the node is not in the TypeScript defs. It should instead return false. (And this should be done without try/catch, e.g. by detecting something is undefined instead of accessing a property of undefined.)

Reply:
Let's put this back to the original version? If we think the other order is better, that should be a separate PR.